shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/Graphite.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf;
+
+import javax.annotation.PostConstruct;
+import javax.validation.constraints.Pattern;
+
+import org.apache.hadoop.fs.Path;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "graphite")
+public class Graphite {
+
+  private boolean enabled = false;
+  private Path config;
+  @Pattern(regexp = "[^:]+:[0-9]+")
+  private String host;
+  private String prefix;
+  private String namespace;
+
+  @PostConstruct
+  void init() {
+    if (config != null || host != null || prefix != null || namespace != null) {
+      enabled = true;
```

Comment:
If we have a specific enabled flag, it should be non optional and this code should go as it would override the value

Reply:
Yeah, I think this is just legacy from how we did it in CT. We could decide to not have it here and just pass the values on to CT and let it figure it out? Not sure we'd ever want to set enabled to false here if we have actual values set?

Reply:
This is how you configure it in SY right, and then we take those values and write it out to CT?
Now that I think about it I think we should maybe do this differently. Just have a yaml with CT config in SY and link to that when we start up CT. No need for all these classes and copying variables.
It will also be easier if we want to add SY graphite config (for SY reporting) it will be less confusing.